### ISSUE TITLE
Использование SDK-JS в TypeScript

### ISSUE THREAD

**Issue by informixter** [Scorocode/scorocode-SDK-JS#2]
Столкнулся, что нет возможности в подключить стандартными средствами SDK к TypeScript. Как быть подскажите?

**Comment by PROFITVenchurs** [comment]
Здравствуйте. А как подключали?
Мы сами не пробовали, но с ходу удалось найти две ссылки на stackoverflow:
http://stackoverflow.com/questions/36309361/using-npm-modules-in-typescript
http://stackoverflow.com/questions/38224232/how-to-consume-npm-modules-from-typescript

**Comment by PROFITVenchurs** [comment]
Конечно, это может быть не так удобно, но вы всегда можете работать без SDK, по HTTP API
http://scorocode.github.io/scorocode-docs/httpapi/appapi/

**Comment by informixter** [comment]
Пробовал через стандартный импорт
`import * as Scorocode from "scorocode";`
`const Scorocode: any = require("scorocode")`
`import Scorocode = require("scorocode")`

Метод что предлагается в ссылке http://stackoverflow.com/questions/36309361/using-npm-modules-in-typescript не завел

Вторая ссылка http://stackoverflow.com/questions/38224232/how-to-consume-npm-modules-from-typescript тоже не завелась, может мало ковырялся

Как я понял нужно создать файл типов, по аналогии с http://stackoverflow.com/questions/34660265/importing-lodash-into-angular2-typescript-application

ЗЫ TypeScript ^2.0

**Comment by PROFITVenchurs** [comment]
Ок, поставили в TODO разработку файла типов. Быстро не обещаем, но сделаем.

**Comment by ElnaraGainanova** [comment]
а как прописывать ключи приложения? например при использовании angular2

**Comment by nkschmidt** [comment]
Да собственно все тоже самое.
Я к сожалению, не знаком у устройством angular 2, если там есть сервисы, то можно sdk определить как сервис и в момент определения инициализировать с необходимыми ключами.
